ui/src/hooks/useLastBets.ts

```
@@ -0,0 +1,86 @@
+            .reverse()
+            .filter(action => 
+                action.action === PlayerActionType.BET || 
+                action.action === PlayerActionType.RAISE ||
+                action.action === PlayerActionType.CALL ||
+                action.action === PlayerActionType.SMALL_BLIND ||
+                action.action === PlayerActionType.BIG_BLIND
+            );
+
+        // Get the last bet action
+        const lastBetAction = betActions[0];
+        const lastBetAmount = lastBetAction && lastBetAction.amount 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)